modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/CatalogUtils.java

```diff
@@ -41,6 +48,43 @@ public static TableDescriptor fromParams(int id, CreateTableParams params) {
         );
     }
 
+    /**
+     * Converts CreateIndex command params to hash index descriptor.
+     *
+     * @param id Index id.
+     * @param tableId Table id.
+     * @param params Parameters.
+     * @return Index descriptor.
+     */
+    public static IndexDescriptor fromParams(int id, int tableId, CreateHashIndexParams params) {
+        return new HashIndexDescriptor(id,
+                params.indexName(),
+                tableId,
+                true,
```

Comment:
this actually should be `false`. The only valid case of creation of unique hash index is implicit creation of primary key